### PR TITLE
Add more detail to docs

### PR DESCRIPTION
**Pull request by rosteen**
Not ready for merge yet, but getting there.

### CODE REVIEW

**Comment by rosteen** [comment]
@duytnguyendtn In the interest of getting this in sooner, I think an example notebook can be a separate PR. Ready for review.

File: docs/quickstart.rst
Comment (duytnguyendtn) [inline docs/quickstart.rst:37]: Pardon if I got the order wrong, but I think it would help if we mirror the labels we use in our docstrings: 
https://github.com/spacetelescope/astrogrism/blob/main/astrogrism/HST/transform_models.py#L60-L73

```suggestion
This returns a tuple of values (x, y, x0, y0, order), where (x, y) denotes 
coordinates on the direct image, and (x0, y0) denotes the dispersed coordinates on 
```

**Review by duytnguyendtn — APPROVED** [review]
I believe the language in this PR is entirely sufficient for a quickstart documentation for our users and will be very useful for us to start getting astrogrism into the hands of our customers; thanks @rosteen  for grabbing this effort!

Weird behavior I'm seeing that I *think* is just my setup, but there are a few inconsistencies I'm seeing when I build everything from scratch:
1. The `Geometric Transforms` page is missing in the sidebar from all submenus
2. The Class Inheritance Diagram is not rendering properly
3. I'm getting A LOT of missing fonts, the favicon, and other files that should be found in `_static`

I'm not going to block this for the PR merge, so long as we inspect it once it builds before we release. I also have one small language suggestion to consider before merging.

File: docs/quickstart.rst
Comment (rosteen) [reply docs/quickstart.rst:37]: Good call. I think this might apply one other place, I'll double check.

**Pull request merged**
